### DISCUSSION TITLE
PR template in multiple repositories

### DISCUSSION THREAD

**Discussion by stergiosmaster** [community/community#53225] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Hi, Is there a way to create one single pr template and use it for multiple repositories? Copying it to all of them is not sustainable as it needs to be changed in all repositories everytime a change is needed.
Same question about github action for PR name validation

**Comment by bbkx226** [comment, marked as answer]
If you wish to establish a default pull request template for all the repositories within your organization, you may create a repository named .github. The link below provides guidance on how to create organization-wide community health files, which includes creating a default pull request template:

https://github.blog/changelog/2019-02-21-organization-wide-community-health-files/

However, note that it is not currently possible to limit the sharing of this template to only specific repositories, except through the use of external tooling such as git-submodules.

**Reply by stergiosmaster** [reply]
Thank you this is exactly what I was searching. 
Is there something similar for github actions? I want to check the PR naming for example , if it is following rules. Or for checking if there are pull requests that are open for too long. I use github actions for that but still I need to do that in all of my repositories

**Reply by bbkx226** [reply]
It appears that GitHub Actions does not currently support the reuse of workflows.

**Reply by dej10** [reply]
updated [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file) , the one in answer leads to a blank page for me
